### PR TITLE
Updated hierarchy view with correct displays

### PR DESCRIPTION
**Pull request by khuntetad**
The new hierarchy clearly displays the folders and subitems with padding. In order to fix this, the mask containing the text was expanded, and a content size fitter was implemented in order to fix the scaling. A before and after of the hierarchy view can be found below.

Before:
Screenshot (1)

After:
Screenshot (2)

AARD-1263
[Add Hierarchy view back to Scene](https://jira.autodesk.com/browse/AARD-1263)

### CODE REVIEW

**Review by matthew17754 — APPROVED** [review]

File: engine/Assets/Scripts/UI/Hierarchy/Hierarchy.cs [hunk 1]
```diff
@@ -26,19 +26,21 @@ public class Hierarchy : MonoBehaviour
...
 
-        public void Start() {
+        public void Start()
+        {
 
             // ContextMenu.Show()
             // ContextMenu.Show(new Vector2(500, -500), "Test Menu", new string[]{"Hello", "There"});
 
-            rootFolder.Init("Scene", null);
+            rootFolder.Init("Scene ajsdf;lajsd;lfajs", null);
```
Comment (KyroVibe) [inline engine/Assets/Scripts/UI/Hierarchy/Hierarchy.cs:40]: Rename all the items back to their original names

File: engine/Assets/Scripts/UI/Hierarchy/Hierarchy.cs [hunk 2]
```diff
@@ -26,19 +26,21 @@ public class Hierarchy : MonoBehaviour
...
 
-            var robots = rootFolder.CreateFolder("Robotssssssss");
-            var fields = rootFolder.CreateFolder("Fields");
+            var robots = rootFolder.CreateFolder("Robotssssssss j;lka");
+            var fields = rootFolder.CreateFolder("Fields ajsdflajs;dlf ");
 
             robots.CreateItem("997 Spartan Robotics");
```
Comment (KyroVibe) [inline engine/Assets/Scripts/UI/Hierarchy/Hierarchy.cs:45]: Get rid of the ErrorCode and Spartan Robotics example items

File: engine/Assets/Scripts/HierarchyTest.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using Synthesis.UI.Hierarchy;
+using Synthesis.UI.Hierarchy.HierarchyItems;
+
+public class HierarchyTest : MonoBehaviour
```
Comment (KyroVibe) [inline engine/Assets/Scripts/HierarchyTest.cs:8]: Remove this test script. It doesn't work

**Review by KyroVibe — CHANGES_REQUESTED** [review]
Move the hierarchy to be closer to the side of the screen and move it a bit down to make it look a bit nicer

**Review by KyroVibe — APPROVED** [review]
Looks good, nice work pushing through the insanity we call Unity

**Pull request merged**